models/lease.py

```diff
@@ -26,6 +27,10 @@ def json(self):
             "unitNum": self.unitNum,
         }
 
+    def is_active(self):
+        now = datetime.now()
+        return now > self.dateTimeStart and now < self.dateTimeEnd
+
```

Comment:
How is this different from the existing `active` method? Do we need both?

Reply:
oh, I realized this is an instance method, and the other one is a class method. I'm still curious if we need both of them. I think we could get all the active leases by doing something like:
`active_leases = self.leases.filter(LeaseModel.active())`
And from all the active_leases, pull all the tenant_ids out into an Array.

And then ask the database to retrieve all tenants with those ids.
TenantModel.query`.get_by_ids`(tenant_ids) <- I'm sure they have a method for this.